modules/navier_stokes/examples/flow-over-circle/mesh.i

```diff
@@ -55,6 +55,7 @@
     inputs = 'final_ccmg left right'
     stitch_boundaries_pairs = 'left right; right left'
     clear_stitched_boundary_ids = false
+    prevent_boundary_ids_overlap = false
```

Comment:
Haha I thought that @AlbertJP7 and I had run into this last summer

Reply:
yeah and you worked around it so now it fails ^^

To be fair it s very easy to run into it when using moose generated meshes. And stuff like 'left' and 'right' should really be cleared with clear_stitched_boundary_ids = false when stitching